code/modules/mob/status_procs.dm

```diff
@@ -21,7 +21,17 @@
 
 ///Blind a mobs eyes by amount
 /mob/proc/blind_eyes(amount)
-	adjust_blindness(amount)
+	blindness(amount)
+
+/**
+  * Sets a mob's blindness to an amount if it was not above it already, similar to how status effects work 
+  */
+/mob/proc/blindness(amount)
```

Comment:
If it is only used in one place, five lines above, why should it have its own proc?